tests/animalShelter.test.ts

```diff
@@ -115,4 +115,24 @@ describe("Animal Shelter Tests", () => {
     expect(animals.mamamls).toBe(3);
     expect(animals.reptiles).toBe(2);
   });
+  test("should succesfully set shelter trritory", () => {
+    const animalShelter = AnimalShelter.getInstance();
+    animalShelter.setShelterTerritory(2000);
```

Comment:
extract 2000 in constant

Reply:
do you happen to test for a territory of 0 or negative ?

Reply:
why do i need to extract a number in the tests? Following this logic i need to extract every number in the test folder